adapter/internal/adapter/adapter.go

```diff
@@ -236,7 +237,7 @@ func Run(conf *config.Config) {
 		go messaging.ProcessEvents(conf)
 
 		// Fetch APIs from control plane
-		fetchAPIsOnStartUp(conf)
+		fetchAPIsOnStartUp(conf, nil)
```

Comment:
This should be fetched if the eventhub is enabled but ga is disabled. Otherwise the startup fetch would be called twice.
(Since the JMS connection for subscription/ application data is not moved to the global adapter yet.

Reply:
Fixed with 36241ea23d0be9e624f45a5d29539f058f22998a